deployer/deployer.go

```diff
@@ -1070,7 +1070,7 @@ func postCommandOrcs(
 	}
 
 	req.Header.Add("Content-Type", "application/json")
-	req.SetBasicAuth("shpandrak", "1234")
+	req.SetBasicAuth("admin", "nazgul")
```

Comment:
Wouldn't you be better off defining these constants once and use those in all places?

Reply:
Yes it will, fixing